### DISCUSSION TITLE
Performing openssl cmp IR using server address in IPV6 format

### DISCUSSION THREAD

**Discussion by sumanth797** [openssl/openssl#24390] [Q&A] [answered]
Hi,
I'm trying to perform OpenSSL CMP IR using server address in **IPV6** format. So, when I do it is not able to parse the IP, do we have such option where I can pass IPV4/ IPV6 address format?

openssl cmp -cmd ir **-server 2a00:9a00:4000:8::26:7:8081**/pkix/ -recipient /C=CN/O=TEST/CN=ABCD -key key.priv -cert cert.pem -subject /C=CN/O=TEST/CN=ABCD -certout ee_cert.pem -extracertsout trustedextracerts.pem -unprotected_errors
cmp_main:apps/cmp.c:2833:CMP info: using section(s) 'cmp' of OpenSSL configuration file '/etc/pki/tls/openssl.cnf'
cmp_main:apps/cmp.c:2842:CMP info: no [cmp] section found in config file '/etc/pki/tls/openssl.cnf'; will thus use just [default] and unnamed section if present
**setup_client_ctx:apps/cmp.c:1882:CMP error: cannot parse -server URL: 2a00:9a00:4000:8::26:7:8081/pkix/
cmp_main:apps/cmp.c:2962:CMP error: cannot set up CMP context
CMP error: invalid url path
CMP error: error parsing url**

So, how to pass the IPV6 address in the command and also in C++ API. Is there such option provided?
Thanks in advance.

**Comment by t8m** [comment, marked as answer]
Assuming 8081 is port the proper way to specify the url is: `https://[2a00:9a00:4000:8::26:7]:8081/pkix/`

**Comment by sumanth797** [comment]
Thanks, @t8m.
It worked
